### PR TITLE
Meetup migration

### PR DESCRIPTION
**Pull request by danielfone**
Hello! 👋🏻 

This is my resurrection of the old Christchurch Ruby website so that we can migrate off meetup.com. These are the requirements I have worked to:

- Index page has:
  - [x] welcome blurb
  - [x] link to mailing list
  - [x] next event
  - [x] previous events
- Event page has:
  - [x] talk description
  - [x] time and location
  - [x] link to map

The diff is large, but you probably want to review the commits independently since most of the diff is just mechanically importing meetup data in the first commit. The interesting and opinionated changes are in 9648d673630e5664a424f1afe4dc87cf944499d4

I'd welcome PRs that improve the design 🎨 ✨ , but I've spent my creativity budget on just getting the thing working.

## Preview

Home page:
Screenshot 2024-08-27 at 9 28 42 AM
Screenshot 2024-08-27 at 9 33 29 AM

Event page:

Screenshot 2024-08-27 at 9 32 56 AM
Screenshot 2024-08-27 at 9 33 17 AM

### CODE REVIEW

File: _events/2015/2015-02-19-meetup.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+location: Christchurch Office, Catalyst IT Limited, Level 1, 284 Kilmore St, Christchurch
+time: 7:00pm
+title: Monthly Meetup
+
+---
+
+TBA. We'll update these details closer to the time. :)
```
Comment (parndt) [inline _events/2015/2015-02-19-meetup.md:8]:
```suggestion
```

we did not

File: _events/2020/2020-04-16-meetup.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+location: Christchurch Office, Catalyst IT Limited, Level 1, 284 Kilmore St, Christchurch
+time: 7:00pm
+title: Monthly Meetup
+
+---
+
+TBA. We'll update these details closer to the time. :)
```
Comment (parndt) [inline _events/2020/2020-04-16-meetup.md:8]:
```suggestion
```

I am guessing this meetup was cancelled due to COVID-19 restrictions

File: _events/2020/2020-05-21-meetup.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+location: Christchurch Office, Catalyst IT Limited, Level 1, 284 Kilmore St, Christchurch
+time: 7:00pm
+title: Monthly Meetup
+
+---
+
+TBA. We'll update these details closer to the time. :)
```
Comment (parndt) [inline _events/2020/2020-05-21-meetup.md:8]:
```suggestion
```

was this also cancelled due to COVID-19 restrictions?

File: _events/2015/2015-02-19-meetup.md (hunk 1 above)
Comment (danielfone) [reply _events/2015/2015-02-19-meetup.md]: Good call. I missed those on the first pass. Removed now along with the others.

**Review by parndt — APPROVED** [review]
onward!

**Review by patrickdavey — APPROVED** [review]
Amazing work - thank you so much.

**Pull request merged**